### PR TITLE
Update readme with installation instructions and updated URL and info.

### PR DESCRIPTION
**Pull request by shadyvb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,48 @@ openSSL module as extension for chassis.io. See https://github.com/Chassis/Chass
...
+- Drag the certificate into the "System" keychain.
+- Right-click it and click "Get Info".
+- Expand the "Trust" section if it's not already.
+- Under "Secure Sockets Layer (SSL)", select "Always Trust".
+- Close the window. At this point you may have to enter your Mac account password.
+- Restart your browser for this to take effect.
+
+### IE and Edge on Windows ( [reference](https://medium.com/@ali.dev/how-to-trust-any-self-signed-ssl-certificate-in-ie11-and-edge-fa7b416cac68) )
+
+> - Browse to your page (e.g. https://vagrant.local) in Internet Explorer which should use your self-signed SSL certificate. You should be greeted by an error message saying your certificate is not trustworthy.
```
Comment (roborourke) [inline README.md]: Are these instructions in a blockquote because they're from the above link?

File: README.md [hunk 2]
```diff
@@ -7,8 +7,48 @@ openSSL module as extension for chassis.io. See https://github.com/Chassis/Chass
...
+## Trusting the certificate
+
+In order to avoid security errors and get that nice green padlock in your location bar, you should add the site's SSL certificate to your trust store. The certificate can be found at chassis/{DOMAIN_HERE}.cert .
+
+### macOS via CLI
+`sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain chassis/{DOMAIN_HERE}`
+
+### Firefox on any operating system:
+
+Open Firefox's Preferences.
```
Comment (roborourke) [inline README.md]: Can these steps be a list as well for consistency?

File: README.md [hunk 3]
```diff
@@ -7,8 +7,48 @@ openSSL module as extension for chassis.io. See https://github.com/Chassis/Chass
...
+1. Both the certificate and key are exported to `chassis/` directory, so it can be used with local dev servers, like webpack dev server.
+
+## Trusting the certificate
+
+In order to avoid security errors and get that nice green padlock in your location bar, you should add the site's SSL certificate to your trust store. The certificate can be found at chassis/{DOMAIN_HERE}.cert .
+
+### macOS via CLI
+`sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain chassis/{DOMAIN_HERE}`
+
+### Firefox on any operating system:
```
Comment (roborourke) [inline README.md]: I also found this method for Firefox:

https://github.com/humanmade/altis-local-chassis/tree/master/docs#firefox

- Open Firefox
- Browse to about:config
- Set `security.enterprise_roots.enabled` to true

File: README.md [hunk 4]
```diff
@@ -7,8 +7,48 @@ openSSL module as extension for chassis.io. See https://github.com/Chassis/Chass
...
+> - Browse to your page (e.g. https://vagrant.local) in Internet Explorer which should use your self-signed SSL certificate. You should be greeted by an error message saying your certificate is not trustworthy.
+> - Click “Continue to this website”.
+> - Click on “Certificate error” in the address bar, and then click “View certificates”.
+> - Click “Install Certificate”.
+> - Click “Place all certificates in the following store”, and then click “Browse”. Do not rely on the preselected option to automatically select the certificate store as this will not work!
+> - Inside the dialog box, click “Trusted Root Certification Authorities”, and then click “OK”.
+> - Finish the dialog.
+> - When you get a security warning, click “Yes” to trust the certificate.
+> - Reload your page. The certificate should be working fine now.
+
```
Comment (roborourke) [inline README.md]: Could add Windows command line option too:

https://github.com/humanmade/altis-local-chassis/blob/master/inc/class-command.php#L234

When running the command prompt or some other terminal app as administrator you can do:

```
certutil -enterprise -f -v -AddStore "Root" "vagrant.local.cert"
```

File: README.md (hunk 1 above)
Comment (shadyvb) [reply README.md]: Yup, and because I haven't tested them myself. Looks ugly ? :D

File: README.md (hunk 3 above)
Comment (shadyvb) [reply README.md]: Added as alternative method.

**Comment by shadyvb** [comment]
@roborourke Back at ya!

File: README.md (hunk 1 above)
Comment (roborourke) [reply README.md]: Meh, I'm not too bothered. Just wondered why it was different really! Good to be consistent though.

File: README.md [hunk 5]
```diff
@@ -7,8 +7,61 @@ openSSL module as extension for chassis.io. See https://github.com/Chassis/Chass
...
+1. Both the certificate and key are exported to `chassis/` directory, so it can be used with local dev servers, like webpack dev server.
+
+## Trusting the certificate
+
+In order to avoid security errors and get that nice green padlock in your location bar, you should add the site's SSL certificate to your trust store. The certificate can be found at `chassis/{DOMAIN_HERE}.cert` .
+
+### CLI methods
+
+#### macOS
+`sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain chassis/{DOMAIN_HERE}.cert`
```
Comment (roborourke) [inline README.md]:
```suggestion
```
sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain chassis/{DOMAIN_HERE}.cert
```
```

**Pull request merged**